internal/gke/backup.json

```diff
@@ -0,0 +1,179 @@
+{
```

Comment:
Can you please create subdir `test-fixtures` and move the file there, for better visibility and consistency ?